Shared/UI/Face Detection/ImageCaptureViewController.swift

```diff
@@ -38,27 +38,30 @@ class ImageCaptureViewController: UIViewController, AVCaptureVideoDataOutputSamp
 
     private(set) var cameraType: CameraType = .front
     var flashMode: AVCaptureDevice.FlashMode = .auto
-#warning("Convert to async")
-    func requestAthorization() -> Future<Bool, Never> {
-        return Future { promise in
+
+    func requestAuthorization() async -> Bool {
+        let isGranted: Bool = await withCheckedContinuation { continuation in
             if self.isAuthorized {
```

Comment:
Seems like I was redundant here. isGranted the same as isAuthorized? lets use just one.

Reply:
Probably remove isAuthorized, I could see it getting out of sync.